website_management/views.py

```diff
@@ -152,16 +153,18 @@ def add_new_webpage(request):
 @login_required
 def view_all_webpages(request):
     """display all webpage"""
-    webs = Webpage.objects.all().order_by('id').reverse()
-    context = {'webs': []}
-    for item in webs:
+    filtering = WebpageFilter(request.GET,
+                      queryset=Webpage.objects.all().order_by(
+                          'id').reverse())
+    context = {'webs': [], 'filter': filtering}
+    for item in filtering:#webs:
```

Comment:
__[flake8]__
* [E261] at least two spaces before inline comment
* [E262] inline comment should start with '# '